.vale/fixtures/RedHat/CaseSensitiveTerms/testvalid.adoc

```diff
@@ -253,7 +253,7 @@ Red Hat Virtualization Manager
 RedBoot
 Redis
 RESTEasy
-Role-based access control
+role-based access control (RBAC)
```

Comment:
Perhaps leave out the acronym? It may not be required in all instances in the docs, for example if you use the term infrequently in a module? There's also a separate rule for acronyms I believe

Reply:
@rohennes - Good point :+1: Fixed. Thanks.